code/game/objects/items/weapons/extinguisher.dm

```diff
@@ -106,6 +106,8 @@
 
 		src.last_use = world.time
 
+		if(reagents.chem_temp > 300 || reagents.chem_temp < 280)
+			add_attack_logs(user, null, "Sprayed with superheated or cooled fire extinguisher at Temperature [reagents.chem_temp]K")
```

Comment:
Can include the target here to see which direction they sprayed in in the logs

Reply:
done